server/graphql/v1/mutations/collectives.js

```diff
@@ -90,6 +96,9 @@ export async function createCollective(_, args, req) {
 
     collectiveData.currency = collectiveData.currency || hostCollective.currency;
     collectiveData.hostFeePercent = hostCollective.hostFeePercent;
+    collectiveData.platformFeePercent = isNullOrUndefined(collectiveData.platformFeePercent)
+      ? hostCollective.platformFeePercent
```

Comment:
`hostCollective.platformFeePercent` is not supposed to be set or used, I think we should default on the `OC_FEE_PERCENT` constant (so actually not do anything and remove this line).

Reply:
OK.